ansible/roles/media-licenses/templates/vidis-media-activation-data-sync-cronjob.yml.j2

```
@@ -0,0 +1,96 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  namespace: {{ NAMESPACE }}
+  labels:
+    app: vidis-media-activation-data-sync-cronjob
+    app.kubernetes.io/part-of: schulcloud-verbund
+    app.kubernetes.io/version: {{ SCHULCLOUD_SERVER_IMAGE_TAG }}
+    app.kubernetes.io/name: vidis-media-activation-data-sync-cronjob
+    app.kubernetes.io/component: sync
+    app.kubernetes.io/managed-by: ansible
+    git.branch: {{ SCHULCLOUD_SERVER_BRANCH_NAME }}
+    git.repo: {{ SCHULCLOUD_SERVER_REPO_NAME }}
+  name: vidis-media-activation-data-sync-cronjob
+spec:
+  schedule: 0/5 * * * *
```

Set the shedule variable
```suggestion
  schedule: {{SERVER_VIDIS_SYNC_CRONJOB_SCHEDULE | default("*/7 * * * *", true) 
```